plugins/woocommerce-blocks/bin/build-plugin-zip.sh

```diff
@@ -131,7 +131,7 @@ sh "$SOURCE_PATH/bin/copy-plugin-files.sh" "$SOURCE_PATH" "$SOURCE_PATH/zip-file
 cd "$(pwd)/zip-file"
 if [ $TYPE = 'DEV' ]; then
 	touch blocks.ini
-	printf 'woocommerce_blocks_phase = 3\nwoocommerce_blocks_env = development' > blocks.ini
+	printf 'woocommerce_blocks_env = development' > blocks.ini
```

Comment:
Hmm actually this file isn't relevant anymore is it? we should probably remove it to prevent confusion.

Reply:
Decided to remove this script.